fv3core/stencils/ray_fast.py

```diff
@@ -27,29 +27,28 @@ def dm_layer(rf, dp, wind):
     return (1.0 - rf) * dp * wind
 
 
-@gtstencil()
-def ray_fast_wind(
+def ray_fast_wind_compute(
     u: FloatField,
     v: FloatField,
     w: FloatField,
     dp: FloatFieldK,
     pfull: FloatFieldK,
     dt: float,
     ptop: float,
+    rf_cutoff: float,
     rf_cutoff_nudge: float,
     ks: int,
     hydrostatic: bool,
+    tau: float,
```

Comment:
same comment as for rf_cutoff, can be an external to import rather than a runtime argument